dbt_copilot_helper/commands/dns.py

```diff
@@ -439,14 +440,17 @@ def separate_hyphenated_application_environment_and_service(
     return domain_name, response["LoadBalancers"][0]
 
 
-@click.group()
+@click.group(cls=ClickDocOptGroup)
 def domain():
     pass
 
 
 @domain.command()
 @click.option(
-    "--domain-profile", help="aws account profile name for R53 domains account", required=True
+    "--domain-profile",
+    help="aws account profile name for R53 domains account",
```

Comment:
```suggestion
    help="AWS account profile name for R53 domains; dev or live",
```

Reply:
If we do that we're showing that information in three places, with the first part of the update usage shows like this:

```
Usage: copilot_helper.py domain check-domain --domain-profile (dev|live) 
                                             --project-profile <project_profile> 
                                             --base-domain <base_domain> 
                                             [--env <env>] 

  Scans to see if Domain exists.

Options:
  --domain-profile [dev|live]  AWS account profile name for Route53 domains
                               account  [required]
  --project-profile TEXT       aws account profile name for certificates
                               account  [required]
  --base-domain TEXT           root domain  [required]
  --env TEXT                   AWS Copilot environment name
  --help                       Show this message and exit.
```

Reply:
Ahhh, yes, of course. It shows the choice in the options bit. 👍